myPackage/Player.java

```diff
@@ -1,14 +1,27 @@
 package myPackage;
 
+
 public class Player {
+    private int playerId;
     private String name;
     private boolean isBatsman;
     private boolean isBowler;
+    private boolean isWicketKeeper;
+    private int teamId;
+    private String teamName;
 
-    public Player(String name, boolean isBatsman, boolean isBowler) {
+    public Player(int playerId, String name, boolean isBatsman, boolean isBowler, boolean isWicketKeeper, int teamId, String teamName) {
```

Comment:
Can we use getter setter here instead of such big constructor. Also analyse Builder pattern for defining values.